src/oph/ehoks/oppijaindex.clj

```diff
@@ -336,7 +336,20 @@
 (defn- opiskeluoikeus-tila-inactive? [tila]
   (some #(= tila %) ["valmistunut"
                      "eronnut"
-                     "katsotaaneronneeksi"]))
+                     "katsotaaneronneeksi"
+                     "peruutettu"]))
+
+(defn opiskeluoikeus-active?
+  "Checks if the given opiskeluoikeus is still valid, ie. not valmistunut,
+  eronnut, katsotaaneronneeksi.
+  TODO: This function can be replaced with the function below once
+  :prevent-finished-opiskeluoikeus-updates? is removed or changed to true."
+  ([opiskeluoikeus-oid]
+    (let [opiskeluoikeus (k/get-opiskeluoikeus-info opiskeluoikeus-oid)]
+      (if-not (opiskeluoikeus-tila-inactive?
+                (get-opiskeluoikeus-tila opiskeluoikeus))
+        true
+        false))))
```

Comment:
Ehkä voisi vaan suoraan palauttaa
`(not (opiskeluoikeus-tila-inactive? (get-opiskeluoikeus-tila opiskeluoikeus)))`